focus-android/app/build.gradle

```diff
@@ -24,7 +24,7 @@ android {
     defaultConfig {
         applicationId "org.mozilla"
         minSdkVersion config.minSdkVersion
-        targetSdkVersion config.targetSdkVersion
```

Comment:
Assuming we're going to update this `config.targetSdkVersion` when we're done targetting AC/Fenix too?